dev-docs/source/GitWorkflow.rst

```diff
@@ -85,6 +85,7 @@ When creating a pull request you should:
 
   - The title should **not** contain the issue number
 - `Reference the issue which the pull request is closing  <https://github.com/blog/1506-closing-issues-via-pull-requests>`_, using one of `these <https://help.github.com/articles/closing-issues-via-commit-messages>`_ keywords
+- State the user who initiated the original issue, if they are named in the issue
```

Comment:
In the talk you mentioned email addresses - would it be worth just putting a little note here to remind developers to obfuscate email addresses in some way so we don't open up scientists to webcrawlers?

Reply:
Good point, thanks @josephframsay .